seqr/migrations/0066_family_post_discovery_mondo_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.23 on 2024-05-22 15:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seqr', '0065_family_external_data'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='family',
+            name='post_discovery_mondo_id',
```

Comment:
do we have to support multiple `post_discovery_mondo_ids` for one family?

Reply:
It was not in the ticket, so I did not build it. I guess probably in the future we may need to, but also at some point in the future we are going to like ingest MONDO and actually have users pick options from a dropdown instead of storing free text, so it seems like the simplicity of maintaining a single field for now is easier, and we can update it to a multiple field at some point in  the future when we do a full overhaul of the field

Reply:
ah ok I was confused about task # 3 (Accommodate reporting for cases with multiple findings across OMIM and MONDO (similar to current logic of multiple findings with multiple OMIMs)) but that seems to be related to the changes in [anvil_metadata_utils.py](https://github.com/broadinstitute/seqr/pull/4105/files#diff-fcba4e6ac6983d5e2f987e729d8b190d3add7e8dd3b75a59b9c274bbdadd7ce6)